### PR TITLE
ENH: Make more use of conda-forge.yml

### PR DESCRIPTION
**Pull request by ericdill**
These are the changes that I needed to make to get repos correctly configured to build linux and osx conda binaries.  Thoughts?

### CODE REVIEW

**Comment by pelson** [comment]
Thanks @ericdill.

I'll take a good look at these changes in a few days - I'm currently desperately trying to put together a presentation for tomorrow :wink:

File: conda_smithy/templates/travis.yml.tmpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-{% set conda_vars = {'python': 'CONDA_PY', 'numpy': 'CONDA_NPY'} %}
+{% set conda_vars = {'python': ' CONDA_PY', 'numpy': 'CONDA_NPY'} %}
```
Comment (scopatz) [inline conda_smithy/templates/travis.yml.tmpl]: Why is there a space here?

Comment (ericdill) [inline conda_smithy/templates/travis.yml.tmpl]: Without the space travis was not correctly identifying the build matrix.

[build #9] (https://travis-ci.org/scikit-xray/scikit-xray-feedstock-dev/builds/82626440) Without space there is only one build
[build #10] (https://travis-ci.org/scikit-xray/scikit-xray-feedstock-dev/builds/82627095) With the space there are two builds
[diff between build #10 and build #9] (https://github.com/scikit-xray/scikit-xray-feedstock-dev/compare/ecc4e78a2966...975967fe6c9f)

Comment (scopatz) [inline conda_smithy/templates/travis.yml.tmpl]: I see, I think instead line 17 of this file should be changed, though.

Comment (scopatz) [inline conda_smithy/templates/travis.yml.tmpl]: Namely it should be 

```
 - {{ matrix_env(case) }}
```

That is, remove the second minus.

Comment (ericdill) [inline conda_smithy/templates/travis.yml.tmpl]: I agree that this is a better solution. Thanks.

**Comment by scopatz** [comment]
Ok!  This looks good to me.

**Comment by scopatz** [comment]
Thanks @ericdill

**Comment by pelson** [comment]
Cool. Thanks @ericdill, and thanks @scopatz for reviewing.

We should try to keep track of who and where conda-smithy is being used - I've started a wiki page, so please feel free to add to it: https://github.com/conda-forge/conda-smithy/wiki

**Comment by ericdill** [comment]
I am :+1: on keeping track of where conda-smithy is being used. I'll keep that page up to date as I bring more of our repos online with conda-smithy.

**Pull request merged**